.gitignore

```diff
@@ -0,0 +1,7 @@
+*.gcompcache
+*.lvprojectcache
+*.niaddoncache
+*.lvcodedb
+*.aliases
+*.lvlps
+*.gcompcache-journal
```

Comment:
Might want to add `Builds` and `*.lvcompile`

Would be great to get these to build on Travis and publish to gh-pages some day, but not today :)